frontend/src/App.vue

```diff
@@ -1,19 +1,30 @@
 <template>
   <main>
     <wof-header/>
+    <wof-small-error size=1>Error description</wof-small-error>
     <div class="main-wrapper">
-      <router-view msg="This is a basic frontend"></router-view>
+      <router-view msg="This is a basic frontend">
+      <wof-small-error size=1>An element with position: fixed; is positioned relative to the viewport, which means it always stays in the same place even if the page is scrolled. The top, right, bottom, and left properties are used to position the element.
```

Comment:
That line needs breaks.

Reply:
Is there a reason there are none?

Reply:
Oh, I gues it's for testing only. Then you probably should stick this code in a comment on not include it in the PR